lib/urls.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
   },
   establishment: {
     dashboard: '/establishments/:establishmentId',
-    read: '/establishments/:establishmentId/details'
+    read: '/establishments/:establishmentId/details',
+    asru: '/establishments/:establishmentId/asru/:asruUser',
+    inspectors: '/establishments/:establishmentId/asru/inspectors',
+    spocs: '/establishments/:establishmentId/asru/spocs'
```

Comment:
We seem to have the same URL defined 3 times here. If we're mounting onto the parameterised URL then any link builders should use the parameterised version, with the parameter defined.

Reply:
done